Assets/Scripts/PlayerBehaviour.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerBehaviour : MonoBehaviour
+{
+    private GameObject otherPlayer;
+    public float dashSpeed;
+    public float dashHeight;
+    private float playerHeight = 1f;
+
+    public int health = 2;
+    public GameObject fireball;
+    public GameObject shield;
+    public GameObject windslash;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        otherPlayer = GameObject.Find("PlayerCamera").GetComponent<PlayerCamera>().otherPlayer;
```

Comment:
I'm not entirely sure how c# works, does this create direct reference to the camera.otherPlayer or does it create a copy? If its not a direct reference this will need to be updated when the player swaps